### PR TITLE
Refactored converting equity value to float

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/converter/additional_uo.py [hunk 1]
```diff
@@ -61,7 +61,8 @@ def add_founders_info(self, records):
             if founder_equity_info:
                 founder_equity = founder_equity_info[0].text
             if founder_equity:
-                founder_equity = float(founder_equity.replace(',', '.'))
+                founder_equity = founder_equity.replace(',', '.')
+                founder_equity = float(founder_equity)
```
Comment (criticus) [inline business_register/converter/additional_uo.py:65]: have you tested if it works as expected?

Comment (roman-tiukh) [reply business_register/converter/additional_uo.py:65]: what reason of this change?

File: business_register/models/company_models.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ class Company(DataOceanModel): #constraint for not null in both name & short_nam
     name = models.CharField(max_length=500, null=True)
     short_name = models.CharField(max_length=500, null=True)
     company_type = models.ForeignKey(CompanyType, on_delete=models.CASCADE)
-    edrpou = models.CharField(max_length=260)
+    edrpou = models.CharField(max_length=260, db_index=True)
```
Comment (roman-tiukh) [inline business_register/models/company_models.py:20]: is "edrpou" have max_lenght 260? why?

File: business_register/converter/additional_uo.py (hunk 1 above)
Comment (roman-tiukh) [reply business_register/converter/additional_uo.py:65]: this is not a solution of problem

**Pull request merged**